doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -147,16 +204,44 @@ To sign a message using an ED25519 EVP_PKEY structure:
         EVP_MD_CTX_free(md_ctx);
     }
 
+To verify a message using an ED448 EVP_PKEY structure and the EVP_DigestVerify API:
+
+    void do_verify(EVP_PKEY *ed_key, unsigned char *msg, size_t msg_len,
+                   unsigned char *sig, size_t sig_len)
+    {
+        EVP_MD_CTX *md_ctx = EVP_MD_CTX_new();
+
+        const OSSL_PARAM params[] = {
+            OSSL_PARAM_utf8_string ("instance", "Ed448", 5),
+            OSSL_PARAM_octet_string("context-string", (unsigned char *)"A protocol defined context string", 33),
+            OSSL_PARAM_END
+        };
+
+        /* The input "params" is not needed if default options are acceptable.
+           Use NULL in place of "params" in that case. */
+        EVP_DigestVerifyInit_ex(md_ctx, NULL, NULL, NULL, NULL, ed_key, params);
+
+        int ret = EVP_DigestVerify(md_ctx, sig, sig_len, msg, msg_len);
+        /*
+         * ret == 1 indicates success, 0 verify failure and < 0 for some
+         * other error.
+         */
+        ...
+        EVP_MD_CTX_free(md_ctx);
+    }
+
```

Comment:
Please add a HISTORY section that would mention that this is new functionality in 3.3.

Reply:
Also please add a CHANGES.md entry.